Stripe/PublicHeaders/STPCustomer.h

```diff
@@ -14,6 +14,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  *  An `STPCustomer` represents a deserialized Customer object from the Stripe API. You can use `STPCustomerDeserializer` to convert a JSON response from the Stripe API into an `STPCustomer`.
+ *  TODO: deprecate
```

Comment:
Ah, this TODO is wrong – we should just deprecate the `customerWithStripeID:defaultSource:sources:` constructor. I'll update.